Gum/Commands/EditCommands.cs

```diff
@@ -496,6 +496,7 @@ public void CreateComponent()
                     if (string.IsNullOrEmpty(whyNotValid))
                     {
                         ComponentSave componentSave = new ComponentSave();
+                        componentSave.BaseType = "Container";
```

Comment:
This line fix the issue of the component save file to not include a lot of variables (so the preview could not display the instances)